project/Dependencies.scala

```diff
@@ -29,7 +29,8 @@ object Dependencies {
     "com.nrinaudo"                %% "kantan.csv"                 % "0.5.1",
     "com.miguelfonseca.completely" % "completely-core"            % "0.8.0",
     "org.jsoup"                    % "jsoup"                      % "1.13.1",
-    "org.webjars.npm"              % "govuk-frontend"             % "3.11.0"
+    "org.webjars.npm"              % "govuk-frontend"             % "3.11.0",
+    "com.openhtmltopdf"           % "openhtmltopdf-pdfbox"      % "1.0.10"
```

Comment:
Please run `scalafmtSbt` to fix formatting

Reply:
Done